vizro-core/src/vizro/_vizro.py

```diff
@@ -16,25 +16,35 @@
 class Vizro:
     """The main class of the `vizro` package."""
 
-    _user_assets_folder = Path.cwd() / "assets"
-    _lib_assets_folder = os.path.join(os.path.dirname(os.path.abspath(__file__)), "static")
-
-    def __init__(self):
+    def __init__(self, **kwargs):
         """Initializes Dash."""
-        _js, _css = _append_styles(self._lib_assets_folder, STATIC_URL_PREFIX)
-        self.dash = dash.Dash(
-            use_pages=True,
-            pages_folder="",
-            external_scripts=_js,
-            external_stylesheets=_css,
-            assets_folder=self._user_assets_folder,
+        self.dash = dash.Dash(**kwargs, use_pages=True, pages_folder="")
```

Comment:
Are we allowing `**kwargs` now so that users can set things like `url_base_pathname`? And what if for some reason they want to provide one of the two other arguments `use_pages` and `pages_folder`. Should we not intercept this with a custom error message?

Reply:
Yh that was the case I was thinking off. It reminds me of the old decorator days, and struck me as an unintuitive error message because they did not provide the same keyword twice. Agree though that it is very unlikely, but maybe let's settle once and for all the policy on this case.

And alternative would be `setdefault` no? In fact, should this not be the path forward, as we do that similarly in table?

```py
kwargs = kwargs.setdefault("use_pages",True)
kwargs = kwargs.setdefault("pages_folder","")
```

Reply:
This wouldn't make sense here because Vizro relies very heavily on Dash pages e.g. if you set `use_pages = False` then nothing will work at all. Overriding these options should not be possible.

If you think it's helpful then I can put in something which checks if the user provided `use_pages` or `pages_folder` and emits a different exception that explains it's invalid, but honestly I think this case is so unlikely to occur that it's not worth it.

Reply:
Hm ok, then ignore this one. I suppose someone every attempting to change those should actually be clever enough to read the source code